frontend/src/scenes/settings/organization/OrgEmailPreferences.tsx

```diff
@@ -19,6 +19,7 @@ export function OrganizationEmailPreferences(): JSX.Element {
             checked={!!currentOrganization?.is_member_join_email_enabled}
             disabled={isRestricted || !currentOrganization}
             label="Email all current members when a new member joins"
+            disabledReason={!isRestricted ? 'You need to be an admin or owner to change this setting.' : undefined}
```

Comment:
Hey @raquelmsmith – after testing this out I realized that I think the logic is reversed. This condition should have matched the `disabled` condition above. Currently this is making it so the toggle is _always_ disabled (because one of `disabled` or `disabledReason` will be true and the other will be false/undefined).
```suggestion
            disabledReason={isRestricted ? 'You need to be an admin or owner to change this setting.' : undefined}
```